docs/interpreting-experiments/confidence-intervals.md

```diff
@@ -1,34 +1,266 @@
+
+:::info
+In general, a positive lift will be *good* (colored green) and a negative lift will be *bad* (colored red). However, for metrics such as page load time or app crashes, a higher number is *bad*. If you've set the "Desired Change" field in the fact definition to "Metric Decreasing", then positive lifts will be in <span class="negative-change-red-bg">red</span> and negative lifts will be in <span class="positive-change-green-bg">green</span>.
+:::
 
 ![Confidence intervals](../../static/img/measuring-experiments/confidence.png)
 
-### How we calculate the confidence intervals
-The conventional way of calculating confidence intervals are based on the Z-test (assuming that there is sufficient data for CLT). That is, a confidence interval for unknown mean $\mu$ based on estimator based on estimator $\hat\mu_n = \bar X$ is found by $(\hat\mu_n - z_{\alpha/2} \hat{\text{SE}},  \hat\mu_n + z_{\alpha/2} \hat{\text{SE}})$, where for $\alpha = 0.05$, $z_{\alpha / 2} = 1.96$. Under the usual assumptions of classical statistical theory, we then have a guarantee that the random interval computed above contains $\mu$ with probability $1-\alpha$.
+[^1]: This is the [frequentist](https://en.wikipedia.org/wiki/Frequentist_inference) interpretation of the significance level. See [below](#bayesian-analysis) for a discussion of what the significance level means if you are using our [Bayesian](https://en.wikipedia.org/wiki/Bayesian_inference) method.
+[^2]: That is, a higher *false positive* or *Type I error* rate.
 
-However, this approach assumes that the sample size is fixed up front, and if this assumption is violated the guarantees for the confidence intervals become invalid.
+## How we calculate the lift estimates and confidence intervals
 
-#### Sequential Analysis
-Eppo uses [sequential analysis](https://arxiv.org/abs/1810.08240) which allows us to compute confidence intervals that are valid across time. More precisely, the probability that the random confidence interval does not contain the unknown mean at any point in time is bounded by $\alpha$. We use the bound in equation (14) of the [reference](https://arxiv.org/abs/1810.08240), and generate confidence intervals by computing
+Eppo provides three methods for estimating the lift and calculating confidence intervals:
 
-$\hat \mu \pm \frac{\sigma}{t} \sqrt{ (t + \rho) \log \left( \frac{t+\rho}{\rho\alpha^2} \right)}$
+1. Fixed-sample frequentist
+2. Sequential frequentist (*the default*)
```

Comment:
Why not put the default as number 1?

Reply:
I put them in the order they are discussed, and I discussed fixed-sample first because it's A) the one people are most likely to have prior experience with, and B) the advantages of sequential (i.e. being able to make a decision at any time) only really make sense when you understand the fixed-sample case. I tried putting sequential first, anyway, but I found it was a lot to discuss the basics of the test (i.e. comparing as if identical) and also highlight the ability to make a decision at any point.